concepticondata/conceptlists.tsv

```diff
@@ -301,3 +301,5 @@ Maciejewski-2016-100	Maciejewski, Greg and Klepousniotou, Ekaterini	2016		100	ra
 Clark-2008-503	Clark, Eddie R.	2008		503	areal, questionnaire	English	Mienic languages	https://inter.payap.ac.th/wp-content/uploads/linguistics_students/Eddie_Clark_Thesis.pdf	Clark2008		The study includes two word lists, this concept list contains the lexical items in Kim Mun in Laos.		
 Clark-2008-441	Clark, Eddie R.	2008		441	areal, questionnaire	English	Mienic languages	https://inter.payap.ac.th/wp-content/uploads/linguistics_students/Eddie_Clark_Thesis.pdf	Clark2008		The study includes two word lists, this concept list contains the lexical items in Kim Mun in Vietnam.		
 Sawka-2019-201	Sawka, Kenneth S. and Mbewe, Christopher and Josephat, Daka and Schwertfeger, Jacob	2019		201	basic, areal, questionnaire	English	Toka-Leya variants and closely related Tonga languages	https://www.sil.org/resources/publications/entry/80878	Sawka2019		This word list was used for a lexicostatistical analysis of Toka-Leya in Zambia. 201 concepts were used to compare 11 languages and varieties, although the authors stated that there were 202 concepts. Almost all of the words were assigned to semantic categories.	44-52	
+List-2018-1105	List, Johann-Mattis and Greenhill, Simon J. and Anderson, Cormac and Mayer, Thomas and Tresoldi, Tiago and Forkel, Robert	2018		1105	annotated	English	Global	http://clics.clld.org	List2018		This is the concept list underlying the second release of the [Database of Cross-Linguistic Colexifications (CLICS2)](http://clics.clld.org) (for the first version see, [List 2014](:bib:List2014). Based on the CLDF framework, 15 data sets were included in the second version of CLICS. The data comprises 1220 language varieties and 2486 distinct Concepticon concept sets. Instead of providing full graphs for the networks, we provide links to the major clusters for each concept in CLICS. The column ``COMMUNITY`` list the most central node in each of the community clusters in CLICS2. The columns ``FAMILY_FREQUENCY``, ``LANGUAGE_FREQUENCY`` and ``WORD_FREQUENCY`` list the number of times a translation for a given concept occurs in the CLICS2 data. The columns ``DEGREE``, ``WEIGHTED_FAMILY_DEGREE``  and ``WEIGHTED_LANGUAGE_DEGREE`` contain the unweighted and weighted degree for each node.		
```

Comment:
"COMMUNITY" is the identifer for the community, so not the central node, the central node is the Central_CONCEPT